bin/v-add-firewall-ipset

```diff
@@ -44,7 +44,11 @@ IPSET_PATH="$HESTIA/data/firewall/ipset"
 
 # Ensure ipset is installed
 if [ -z "$IPSET_BIN" ]; then
-    apt-get --quiet --yes install ipset > /dev/null
+    if [ -f '/etc/redhat-release' ]; then
```

Comment:
Should be safe to be removed all Hestia installs should 

We should add it to the dependencies instead

Reply:
Do you mean RPM spec or the installer bash script? So this should be removed altogether both for debian and rhel from the above file?

Reply:
What about conditional installs? If these are set required in package requirements they will not respect these flag.

https://github.com/hestiacp/hestiacp/blob/main/install/hst-install-debian.sh#L251
https://github.com/hestiacp/hestiacp/blob/main/install/hst-install-debian.sh#L900